compiler_opt/distributed/local/local_worker_manager.py

```diff
@@ -128,8 +128,12 @@ def __init__(self):
       # thread drainig the receive queue
       self._pump = threading.Thread(target=self._msg_pump)
 
+      # Set the state of this worker to "dead" if the process dies naturally.
```

Comment:
"Naturally" meaning because it raised an exception, because that's the only reason it'd happen, correct?

Reply:
Correct